fix: OPTIC-2093: Audio hotkeys for step back/forward missing

bmartel opened the pull request:
This pull request introduces a minor enhancement to the `AudioUltraWithSettings` component by adding two missing hotkey settings for media navigation: stepping backward and stepping forward.

Enhancements to hotkey settings:

* [`web/libs/editor/src/tags/object/AudioUltra/view.tsx`](diffhunk://#diff-89e98192c0f9010208c26dda9f397621af1b42408bf533864896a921080b1d6eR216-R217): Added `stepBackHotkey` (`media:step-backward`) and `stepForwardHotkey` (`media:step-forward`) to the settings object for improved media navigation.

<img width="3008" alt="Screenshot 2025-04-25 at 3 28 12 PM" src="https://github.com/user-attachments/assets/81bfb2b6-0a09-420a-8f7c-89f83356b113" />

bmartel committed:
fix: OPTIC-2093: Audio hotkeys for step back/forward missing

MihajloHoma commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14705067222)

MihajloHoma committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14705067222

MihajloHoma committed:
Merge branch 'develop' into 'fb-optic-2093'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14705067222

bmartel commented on web/libs/editor/src/tags/object/AudioUltra/view.tsx:
```diff
@@ -213,6 +213,8 @@ const AudioUltraWithSettings: FC<AudioUltraProps> = ({ item }) => {
     // @todo this hotkey should be moved from these settings for a more appropriate place;
     // @todo we are planning to have a central hotkeys management, that would be a better option.
     playpauseHotkey: "audio:playpause",
+    stepBackHotkey: "media:step-backward",
+    stepForwardHotkey: "media:step-forward",
```
```suggestion
    stepBackHotkey: "audio:step-backward",
    stepForwardHotkey: "audio:step-forward",
```

bmartel committed:
Update web/libs/editor/src/tags/object/AudioUltra/view.tsx

bmartel committed:
making audio keys in a separate key namespace and applying the targeted key combinations for step backward and forward

bmartel committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-2093

MihajloHoma commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14750132329)

MihajloHoma committed:
Merge branch 'develop' into 'fb-optic-2093'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14750132329

MihajloHoma committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14750132329
